server/src/server.atlas.ts

```diff
@@ -0,0 +1,9 @@
+import mongooseAtlasConnect from './db.atlas.connect';
+import { app as app } from './app';
+
+import {port} from './helpers/variables'; 
+
+
+app.listen(port, () => {
+  mongooseAtlasConnect(port);
+});
```

Comment:
You did the correct thing with the port. But you are not to move the server file @samuko-things please restore it to its previous position. Server files should not be hidden.